make/photon/db/Dockerfile

```diff
@@ -4,6 +4,12 @@ FROM ${harbor_base_namespace}/harbor-db-base:${harbor_base_image_version}
 
 VOLUME /var/lib/postgresql/data
 
+ENV PGBINOLD /usr/local/pg96/bin/
+ENV PGBINNEW /usr/bin
+
+ENV PGDATAOLD /var/lib/postgresql/data/9.6
```

Comment:
how about  `/tmp/pg/v9.6` and `/tmp/pg/v13`?
And things may work with `mv` instead of having to use `-prune ! name` parms.